doc/compile.md

```diff
@@ -44,6 +44,9 @@ After the configuration you need to compile the miner, follow the guide for your
   - there is no *http* interface available if option is disabled: `cmake .. -DMICROHTTPD_ENABLE=OFF`
 - `OpenSSL_ENABLE` allow to disable/enable the dependency *OpenSSL*
   - it is not possible to connect to a *https* secured pool if option is disabled: `cmake .. -DOpenSSL_ENABLE=OFF`
+- `WIN_UAC` will enable or disable the "Run As Administrator" prompt on Windows.
+  - UAC confirmation is needed to use large pages on Windows 7.
+  - On Windows 10 it is only needed once to set up the account to use them.
```

Comment:
Can we compile by default a small additional programm to activate large page support for windows and linux.
In Win10 the PR workflow is

- start the miner once
- edit the config and enable the option
- start miner as admin 
- change config back (for security reason)
- start miner

With a small helper the workflow is:

- start the helper as admin (only once)
- run the miner ( withou admin)

Never the less I would keep the option in the config.

Reply:
But to avoid the issue with the ignored command line parameter switching to a seperate programm to activate huge page support can be easily solves the issues.

Reply:
On Win7  large pages still require admin rights. That's the main problem. I'm not willing to have two versions **and** a separate program.